CONTRIBUTING.md

```diff
@@ -1,35 +1,40 @@
-# Contributing to Direct State Transfer (DST)
+# Contributing to the Hyperledger Lab Perun
 
 Want to contribute? That's great! Any contribution is welcome, e.g.
 documentation, bug reports, feature request, issues, blog posts,
 tutorials, feature implementations, etc. You can contribute code or
 documentation through the standard GitHub pull request model. For large
 contributions we do encourage you to file a ticket in the GitHub issue
-tracking system prior to any development to coordinate with the DST
+tracking system prior to any development to coordinate with the Perun
 development team early in the process. Coordinating up front helps to
 avoid frustration later on.
 
-*Note that this repository only contains the source code of DST. For
-issues / pull requests related only to documentation please refer to the
-[dst-doc](https://github.com/direct-state-transfer/dst-doc) repository.*
+*Note that this repository only contains the source code of the Perun
+Node. For issues / pull requests related only to documentation please
+refer to the
+[perun-doc](https://github.com/direct-state-transfer/perun-doc)
+repository.*
 
 ## Contribution Guideline
 
 * We use the GitHub [issue
-  tracker](https://github.com/direct-state-transfer/dst-go/issues) and
+  tracker](https://github.com/direct-state-transfer/perun-node/issues) and
   this [branching
   model](https://nvie.com/posts/a-successful-git-branching-model/).
 
-* When you want to submit a patch to any DST project, you must [sign a
-  CLA (once) and sign your commits](#sign-the-cla-and-sign-your-work).
+* When you want to submit a patch to any Hyperledger Labs project, you
+  must [sign your commits](#sign-your-work).
 
 * Your contribution must be licensed under the Apache-2.0 license, the
   license used by this project.
 
 * When submitting a pull-request to the project, please note / ensure
   the following:
-    1. If possible, keep the changes small and simple which makes review
-       process easier.
+
+    0. [Update](#maintain-copyright-holder-contributor-list) the
```

Comment:
Should the numbering start from "1" or is "0" used on purpose ?

Reply:
It's on purpose. It's also the step that only needs to performed once.